helm/aladdin-demo/templates/server/deploy.yaml

```diff
@@ -47,71 +45,77 @@ spec:
           - mountPath: /config/uwsgi.yaml
             name: {{ .Chart.Name }}-uwsgi
             subPath: uwsgi.yaml
-{{ if .Values.deploy.withMount }}
-          - mountPath: /home
+          {{ if .Values.deploy.withMount }}
+          - mountPath: /home/app
             name: {{ .Chart.Name }}-server
-            subPath: app
-{{ end }} # /withMount
+          {{ end }}
         envFrom:
           # Load the data from configMap into the runtime environment
           # This allows us to use os.environ["KEY"] to look up variables
           - configMapRef:
               name: {{ .Chart.Name }}
+        {{ if .Values.resourceLimits.enable }}
         resources:
           requests:
-            cpu: 100m
-            memory: 100Mi
+            cpu: {{ .Values.resourceLimits.server.uwsgi.cpu.requests }}
+            memory: {{ .Values.resourceLimits.server.uwsgi.memory.requests }}
           limits:
-            cpu: 200m
-            memory: 200Mi
-        {{ if .Values.app.readiness.use }}
-        # Readiness probe stops traffic to this pod if it is not ready, wait until it is ready
-        readinessProbe:
-          httpGet:
-            path: /ping
-            port: {{ .Values.app.uwsgi.port }}
-          initialDelaySeconds: {{ .Values.app.readiness.initialDelay }}
-          periodSeconds: {{ .Values.app.readiness.period }}
-        {{ end }} # /app.readiness.use
-        # Liveness probe terminates and restarts the pod if unhealthy
-        {{ if .Values.app.liveness.use }}
-        livenessProbe: 
-          httpGet:
-            path: /ping
-            port: {{ .Values.app.uwsgi.port }}
-          initialDelaySeconds: {{ .Values.app.liveness.initialDelay }}
-          periodSeconds: {{ .Values.app.liveness.period }}
-        {{ end }} # /app.liveness.use
-############# nginx server container ########
-## This is a container for an nginx server
-#############################################
-{{ if .Values.app.nginx.use }}
-      # nginx container, only gets created if the app.nginx.use field is true in values.yaml
-      - name: {{ .Chart.Name }}-nginx
-        image: nginx:1.12-alpine
-        ports: 
+            cpu: {{ .Values.resourceLimits.server.uwsgi.cpu.limits }}
+            memory: {{ .Values.resourceLimits.server.uwsgi.memory.limits }}
+        {{ end }} # resourceLimits.enable
+      # This is the container for an nginx server
+      {{ if .Values.app.nginx.use }}
+      - name: nginx
+        image: nginx:1.17-alpine
+        imagePullPolicy: {{ .Values.deploy.imagePullPolicy }}
+        ports:
           - containerPort: {{ .Values.app.nginx.port }}
             protocol: TCP
         volumeMounts:
-          - mountPath: /etc/nginx/
+          - mountPath: /etc/nginx/nginx.conf
             name: {{ .Chart.Name }}-nginx
+            subPath: nginx.conf
```

Comment:
the 1-line bug that caused INFRA-173 to be filed and started all this cleanup !!